Source/WebKit/WebKitSwift/LinearMediaKit/LinearMediaPlayer.swift

```diff
@@ -127,6 +127,9 @@ enum LinearMediaPlayerErrors: Error {
 #endif
         }
     }
+    var enteredFromInline: Bool {
+        get { swiftOnlyData.enteredFromInline }
```

Comment:
Seems like you can remove `enteredFromInline` from `SwiftOnlyData` and make this a stored property.